ui/StatusQ/src/StatusQ/Popups/StatusSearchLocationMenu.qml

```diff
@@ -103,8 +104,10 @@ StatusPopupMenu {
                         value: model.value
                         text: model.text
                         assetSettings.isImage: !!model.imageSource
-                        assetSettings.name: !!model.imageSource ? model.imageSource : model.iconName
+                        assetSettings.name: !!StatusQUtils.Emoji.iconSource(model.imageSource) ?
```

Comment:
Why not check for `!!model.imageSource`?

Reply:
because `imageSource` has always something, either an image or an emoji (or empty). If it's emoji we have to specify it's path using the `iconSource` (because they're stored locally), if that returns undefined, then we either load the remote image or nothing.